Frends.FTP.UploadFiles/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
```

Comment:
I think this file should be in tests-folder, since this is only needed for tests.